File: Module/PlayWays/src/main/java-room-double/com/module/playways/doubleplay/DoubleRoundInfo.java
package com.module.playways.doubleplay;

import com.module.playways.room.song.model.SongModel;

import java.io.Serializable;

public class DoubleRoundInfo implements Serializable {
    protected int roundSeq;
    SongModel mSongModel;

    public int getRoundSeq() {
        return roundSeq;
    }

    public SongModel getSongModel() {
        return mSongModel;
    }

    public void update(DoubleRoundInfo roundInfo) {

    }
}

File: Module/PlayWays/src/main/java-room-double/com/module/playways/doubleplay/DoubleRoundInfo.kt
package com.module.playways.doubleplay

import com.module.playways.room.song.model.SongModel

import java.io.Serializable

class DoubleRoundInfo : Serializable {
    var roundSeq: Int = 0
        protected set
    var songModel: SongModel? = null
        internal set

    fun update(roundInfo: DoubleRoundInfo) {

    }
}
